extensions/sql-bindings/src/services/azureFunctionsService.ts

```diff
@@ -76,6 +78,7 @@ export async function createAzureFunction(connectionString: string, schema: stri
 			if (!selectedBinding) {
```

Comment:
I see a couple other places here where the user can exit the flow early, seems like we'd want to capture those as well